RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -1116,6 +1142,34 @@ private boolean parseSamPluginOutput(String regFilePath, AbstractFile regAbstrac
             for (Map<String, String> userInfo : userSet) {
                 userInfoMap.put(userInfo.get(SID_KEY), userInfo);
             }
+            
+            // New OsAccount Code 
+            OsAccountManager accountMgr = tskCase.getOsAccountManager();
+            HostManager hostMrg = tskCase.getHostManager();
+            Host host = hostMrg.getHost(tskCase.getDataSource(regAbstractFile.getDataSourceObjectId()));
```

Comment:
We should be able to get the host from the DataSource that was passed into RecentActivity. It should have a getHost() method on it.

Reply:
I am sorry, I am not following.  Content doesn't seem to have a getHost method.

Reply:
Right, but DataSource does and it should be a DataSource object that is passed into RecentActivity.